website/docs/05_design/package-scopes.md

```diff
@@ -0,0 +1,44 @@
+# Package Scopes
+
+At Glasskube, we understand that granular management of Kubernetes resources is crucial. That's why we offer two distinct resource types that determine how widely a package can be installed:
+
+-   **Cluster-scoped packages (ClusterPackages):** Installed once per cluster and accessible from all namespaces. Ideal for cluster-wide functionality and shared resources.  
+-   **Namespace-scoped packages (Packages):** Installed in specific namespaces, offering isolation and flexibility. Perfect for application-specific tools or multiple instances of the same package.
+    
+Let's explore when you'd opt for each scope.
+
+### ClusterPackages: Managing Resources at the Cluster Level
+
+ClusterPackages are the prefered when your package:
+
+-   Needs access to all namespaces in your cluster.
+-   Utilizes `ClusterRoles` and `ClusterRoleBindings` for authorization.
+-   Provides functionality or manages resources that are relevant cluster-wide, such as networking, monitoring, or security tools.
+    
+### Packages: Isolation at the Namespace Level
+
+Packages are ideal when you require:
+
+-   Isolation: Your package's functionality is contained within a specific namespace.  
+-   Flexibility: You need to install the same package multiple times, each instance operating independently in its own namespace.  
+-   When the package uses `Roles` and `RoleBindings` for access control.  
+-   Application-specific tools or components that don't need cluster-wide access.  
+
+### Dependencies and Package Scopes
+
+The distinction between ClusterPackages and Packages also impacts dependencies.
+
+-   **ClusterPackages:** If multiple ClusterPackages/Packages depend on another ClusterPackage, they'll share that dependency. This is also the expected behaviour for dependency on operators. 
+-   **Packages:** When a ClusterPackage/Package depends on a Package, that Package is instantiated as a `component` alongside the original package. Each instance of the original package gets its own, isolated instance of the dependent Package.
+    
+### Defining Package Scope: A Note for Package Authors
+
+By default, packages in Glasskube are cluster-scoped. To designate a package as namespace-scoped, include the scope: `Namespaced` key-value pair in your package definition:
```

Comment:
I'd change it. For backwards compatibility reasons not providing a scope will default to a cluster scope, but we highly recommend explicitly defining one.

In addition we should link to the full package manifest reference page: https://glasskube.dev/docs/reference/package-manifest/